docs/src/index.rst

```diff
@@ -3,7 +3,7 @@ CARTA Interface Control Document
 
 :Date: 30 November 2022
```

Comment:
The date also needs to be updated. I really need to look at #71 again to automate this.

Reply:
Data updated.